projects/packages/stats/src/class-wpcom-stats.php

```diff
@@ -231,8 +231,12 @@ public function get_views_by_country( $args = array() ) {
 	public function get_followers( $args = array() ) {
 
 		$this->resource = 'followers';
+		$followers      = $this->fetch_stats( $args );
 
-		return $this->fetch_stats( $args );
+		// run an action which allows us to fire a recommendation.
+		do_action( 'jetpack_followers_fetched', $followers );
```

Comment:
When adding a new hook, you'll want to add a docblock so our parser can automatically add the hook to our [Jetpack Codex](https://developer.jetpack.com/).

Here is an example:
https://github.com/Automattic/jetpack/blob/5dcfe76bb621fecb87673f81b8324a68c9740628/projects/packages/stats/src/class-main.php#L221